### DISCUSSION TITLE
Unable to download for Disney Plus (Disney+)

### DISCUSSION THREAD

**Discussion by bdtwomey** [aclap-dev/video-downloadhelper#539] [Help and Q&A]
### Your question

Trying to rip video from a certain corporation's website is resulting in an error as described below. Any assistance would be amazing! 

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://www.disneyplus.com/play/07286ca0-5b1e-4c2d-b9b2-a0d99df04e53",
  "error": {
    "details": "Error: Native port disconnected",
    "downloadable_id": "downloadable_6748563351895217",
    "error": "coapp_failure",
    "id": "coapp_failure_ad307ba6-74c3-44c9-8c32-fa2d5da7e4ab"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Perhaps #538 applies to you.

Another thought is that the Disney bundle is a paid subscription service.  I do not have & am not going to get such a subscription.  It is possible that the content is protected by DRM.  Search this forum for mention of DRM.  There is an easy way to determine if DRM is involved.  If it is, you can forget about downloading it.
